model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -548,8 +548,11 @@ def _calculate_avg_first_token_latency(self, llm_output: Dict) -> float:
             )
 
         avg_first_token_latency = float(mean(total_first_token_latencies))
+        reduction_factor = float(
+            PerfAnalyzer.llm_metric_table[0][PerfAnalyzer.REDUCTION_FACTOR]  # type: ignore
+        )
 
-        return avg_first_token_latency
+        return avg_first_token_latency / reduction_factor
```

Comment:
Why are we dividing by reduction factor?

Reply:
So, that we store the value as ms (instead of ns).